web/styleguide/components/svg-blob/index.tsx

```diff
@@ -32,19 +32,45 @@ const SvgBlob: React.FC<SvgBlobProps> = React.memo(
     const { position: translation, drag } = useDraggable({
       onDragEnd: imagePositionChanged,
     });
+    const [imageDimensions, setImageDimensions] = useState<{
+      height: number;
+      width: number;
+    }>({ height: 100, width: 100 });
+    const [scaleHeight, setScaleHeight] = useState<number>(imageScale);
+    const [scaleWidth, setScaleWidth] = useState<number>(imageScale);
 
     useEffect(
       function () {
         if (!image) return setImageString(undefined);
         const reader = new FileReader();
-        reader.addEventListener("load", () =>
-          setImageString(String(reader.result))
+        reader.addEventListener('load', () =>
+          setImageString(String(reader.result)),
         );
         reader.readAsDataURL(image);
       },
-      [image]
+      [image],
     );
 
+    useEffect(() => {
```

Comment:
Dette begynner å bli et perfekt tidspunkt for å introdusere custom hook!

Reply:
Enig! Tenkte kanskje det ble litt mye arbeid å skrive om men det er fin lærdom og absolutt noe å bruke tid på

Reply:
Flyttet alt av imagedata-prosess til egen custom hook, simplifya også prosessen litt, ikke behov for å sende så mye data inn i hooksa som jeg gjorde tidligere.